development/webpack/utils/plugins/ManifestPlugin/helpers.ts

```diff
@@ -1,3 +1,5 @@
+import manifestFlags from '../../../../../manifest-flags.json';
```

Comment:
2. Read customized manifestFlags from `manifest-flags.json` so we can override the one from controller (API) - webpack build